docs/troubleshooting.md

```diff
@@ -24,6 +37,79 @@ The Open vSwitch daemon logs for each `antrea-agent` Pod are also stored on the
 persistent storage of the corresponding node (i.e. the node on which the Pod is
 scheduled), under `/var/log/antrea/openvswitch`.
 
+## Accessing the antrea-controller API
+
+antrea-controller runs as a Deployment, exposes its API via a Service and
+registers an APIService to aggregate into the Kubernetes API. To access the
+antrea-controller API, you need to know the address of it and have credentials
+to access it. There are multiple ways you can do it.
+
+### Using antctl
+
+Typically, `antctl` handles locating and authenticating when it runs in an
```

Comment:
what do you mean by "locating" here?

Reply:
updated to "`antctl` handles locating the Kubernetes API server and authenticating when it runs in an environment with kubeconfig set up.", how about it?